rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java

```diff
@@ -847,7 +847,7 @@ public boolean afterLRANotification() {
 
                         response = asyncResponse.get(PARTICIPANT_TIMEOUT, TimeUnit.SECONDS);
 
-                        if (response.getStatus() == 200) {
+                        if (response.getStatusInfo().getFamily() == Status.Family.SUCCESSFUL) {
```

Comment:
The spec requires a 200 OK response

Reply:
ok, let me revisit this

Reply:
you were right, we were returning 204 from non-JAX-RS AfterLRA methods, so I updated rather that.